### DISCUSSION TITLE
[COMPATIBLE HARDWARE] BlueSCSI works with SupraDrive as HDD replacement

### DISCUSSION THREAD

**Discussion by luciust** [erichelgeson/BlueSCSI-v1#278] [Show and tell]
BlueSCSI 1.1 with 50-pin is compatible with SCSI-1 to ACSI controler in SupraDrive for Atari ST family

- directly replaces QUANTUM 120MB drive
- ON and BUSY LED are working as expected, one might check if the 330Ohm resistors are not to big, as they use old LEDs
- allows for 512MB HDD support (usually those controllers don't go too far)
- does not seem to support 1GB HDD tough
- supports 3 different HDD under same controller - as HD00, HD01 and HD02 - possibly more
- tested with ICD Pro
![supra-drive](https://github.com/user-attachments/assets/708414ae-1382-4240-9d42-bce0e23308f6)

**Comment by erichelgeson** [comment]
Awesome, thanks for the report and enjoy!
